openhands-cli/openhands_cli/agent_chat.py

```diff
@@ -37,12 +39,16 @@ def run_cli_entry() -> None:
         EOFError: If EOF is encountered
     """
 
-    conversation = setup_agent()
+    agent_context = AgentContext(
+        system_message_suffix=f"You current working directory is: {WORK_DIR}",
+    )
```

Comment:
Could we place this inside setup_agent itself, instead of making it an arg

Also it seems when loading the agent back in `AgentStore` we'd similarly need to update the agent context